img/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>The Basic Language ogf the Web:HTML</title>
+        </head>
+        <body>
+            <style>
+                body {background: lightgrey;}
+                head {color:bisque;}
+                </style>
+                📘<p><h1><strong>The Code Magazine</p></h1></strong>
+
+<h2><p><b>The Basic Language of the Web: HTML</h2></p></b>
+<img src="laura-jones.jpg" alt="Laura-Jones" width="90" height="80">
+
+<h5><p>Posted by <b>Laura Jones</b> on Monday, June 21st 2027</h5></p>
+<img src="post-img.jpg" alt="screen post" width="650" height="230">
+
+<h5><p>All modern websites and web applications are built using three fundamental technologies: HTML, CSS and JavaScript. These are the languages of the web.</h5></p>
```

Comment:
tags h1, h2, ... h6 are used for titles only.
For simple paragraphs use simple `<p>`